src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -51,7 +51,7 @@
 (defn- channel-chat-item
   [community-id community-color {chat-id :id muted? :muted? :as chat}]
   (let [sheet-content      [actions/chat-actions
-                            (assoc chat :chat-type constants/community-chat-type)
+                            (assoc chat :chat-type constants/community-chat-type :chat-id (str community-id chat-id))
```

Comment:
Hi @ibrkhalil, Thank you very much for fixing this issue.
Please can you clarify, how this fixes the issue?

Reply:
Is id is nil?

Reply:
Yes

Reply:
And why `chat-id` crated by `community-id` and `chat-id` itself?

Reply:
That's how we reference a community channel chat, We concatenate the community-id with the id of that chat.

Reply:
Ok, this change will make sure that, new changes don't break in communities overview screen.

Reply:
Done